File: POO/temas/4/py/4.py
class Frigo:
    def __init__(self):
        self.__color = "blanco"
        self.__puerta = "cerrada"
        self.__temperatura = 20

    def abre(self):
        print( "se está abriendo" )

    def cierra(self):
        print( "se está cerrando" )

    def fija_temperatura(self):
        print( "se cambia la temperatura" )

    def muestra_estado(self):
        print( self.__color )
        print( self.__puerta )
        print( self.__temperatura )


mi_frigo = Frigo()
mi_frigo.muestra_estado()

File: POO/temas/4/js/4.js
class Frigo {
    constructor()
    {
        this._color = "blanco";
        this._puerta = "cerrada";
        this._temperatura = 20;
    }

    abre()
    {
        console.log("se está abriendo");
    }

    cierra()
    {
        console.log("se está cerrando");
    }

    fijaTemperatura()
    {
        console.log("se cambia la temperatura");
    }

    muestraEstado()
    {
        console.log(this._color);
        console.log(this._puerta);
        console.log(this._temperatura);
    }
}


const miFrigo = new Frigo();
miFrigo.muestraEstado();
